.ado/android-pr.yml

```
@@ -18,8 +18,7 @@ variables:
 jobs:
   - job: AndroidRNPR
     displayName: Android React Native PR
-    pool:
-      vmImage: $(VmImageUbuntu)
+    pool: cxe-ubuntu-20-04-small
```

Could we have this also reference $(PoolUbuntu) ?

---

```suggestion
        pool: $(PoolUbuntu)
```
change for consistency